django/utils/translation/__init__.py

```diff
@@ -219,13 +220,13 @@ def deactivate_all():
     return _trans.deactivate_all()
 
 
-def _string_concat(*strings):
-    """
-    Lazy variant of string concatenation, needed for translations that are
-    constructed from multiple parts.
-    """
-    return ''.join(force_text(s) for s in strings)
-string_concat = lazy(_string_concat, six.text_type)
+def string_concat(*strings):
+    from django.utils.text import format_lazy
+    warnings.warn(
+        'django.utils.translate.string_concat is deprecated. '
```

Comment:
The wording we usually use is "django.utils.translate.string_concat() is deprecated in favor of django.utils.text.format_lazy()."